### PR TITLE
New deps and use the common build files

### PR DESCRIPTION
**Pull request by dcristoloveanu**
New deps and use the common build files

### CODE REVIEW

File: tests/tarray_int_reals/real_tarray_undo_op.c [hunk 1]
```diff
@@ -21,5 +21,6 @@ THANDLE_LL_TYPE_DEFINE_WITH_MALLOC_FUNCTIONS(TARRAY_TYPEDEF_NAME(real_UNDO_OP),
 
 #define realloc_2 real_gballoc_ll_realloc_2
 #define malloc_2 real_gballoc_ll_malloc_2
+#define free real_gballoc_ll_free
 TARRAY_LL_TYPE_DEFINE(real_UNDO_OP, UNDO_OP);
```
Comment (jebrando) [inline tests/tarray_int_reals/real_tarray_undo_op.c:26]: Nitpick: empty spec is unnecessary.  /consider

**Review by jebrando — DISMISSED** [review]
:shipit:

**Comment by anporumb** [comment]
> 

?

---
Refers to: CMakeLists.txt:126 in ada335a. [](commit_id = ada335a42ebcfe48e7955daf70dcad6625a33da1, deletion_comment = False)

File: CMakeLists.txt [hunk 2]
```diff
@@ -46,6 +46,11 @@ if ((NOT TARGET c_build_tools) AND (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/c-buil
     set_default_build_options()
 endif()
 
+if ((WIN32) AND ("${GBALLOC_LL_TYPE}" STREQUAL "JEMALLOC"))
+    # Bring in vcpkg
+    use_vcpkg(${CMAKE_CURRENT_LIST_DIR}/deps/vcpkg)
+endif()
```
Comment (anporumb) [inline CMakeLists.txt:52]: shouldn't this be done by c-pal subrepo?
or it be part of "canon" way of including c-pal?

**Review by anporumb — APPROVED** [review]
:shipit:

File: tests/tarray_int_reals/real_tarray_undo_op.c (hunk 1 above)
Comment (dcristoloveanu) [reply tests/tarray_int_reals/real_tarray_undo_op.c:26]: I assume it is about empty spaces, removed some empty lines.

**Review by jebrando — APPROVED** [review]
:shipit:

**Pull request merged**